CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -559,11 +559,6 @@ TrackerMap::~TrackerMap() {
     TmPsu *psu = ipsu->second;
     delete psu;
   }
-
-  gROOT->Reset();
-
-  //for(std::vector<TColor*>::iterator col1=vc.begin();col1!=vc.end();col1++){
-  //     std::cout<<(*col1)<<std::endl;}
```

Comment:
This change was already part of #41289. I'd suggest to rebase on top of a recent IB.

Reply:
Thanks, I have tried to squash some commits and there were some issues which I am trying to fix. This was one of them.